### PR TITLE
Urls checker

### PR DESCRIPTION
**Pull request by RobertMrowiec**

### CODE REVIEW

File: frontend_developer/react/03_level_3.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Knows how React compares to other frameworks, and how JSX works internally.
 
 ### 🎓 Learn
 
-- 📗 [Angular vs React vs Ember vs Vue.js](http://voidcanvas.com/angular-vs-react-vs-ember-vs-vue-js/)
+- 📗 [Angular vs React vs Ember vs Vue.js](http://voidcanvas.com/angular-vs-react-vs-ember-vs-vue-js)
```
Comment (PoslinskiNet) [inline frontend_developer/react/03_level_3.md:33]: This link is pretty out of date by now. We need more generic comparison I think that will last for longer. @pokiujf @RobertMrowiec

File: index.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+    linkify.find(file).filter(url => url.type === 'url' && ( url.value.includes('http') || url.value.includes('www') )).map(urls => URLs.push({
+      file: fileName,
+      href: urls.value
+    }))
+  }
+
+  console.log('Finished: 0%')
+
+  for (url of URLs) {
```
Comment (mikoscz) [inline index.js:28]: You could leverage some king of the parallel map like a solution here to speed up the whole process. I did some quick test:
current solution: `Duration time:  195.72s`
map:                    `Duration time:  6.47s`.
Which is over 30x faster for the CI to get the feedback. For my test, the output results were the same but maybe there are some edge cases.

File: index.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
+const fs = require('fs');
```
Comment (mikoscz) [inline index.js:1]: From where the deps come?

**Comment by PoslinskiNet** [comment]
@RobertMrowiec I' merging it for now since it already delivers the business value. Please handle any further optimization in a separate PR.
